lib/webpack/rails/manifest.rb

```diff
@@ -54,8 +54,8 @@ def load_manifest
 
         def load_dev_server_manifest
           http = Net::HTTP.new(
-            "localhost",
-            ::Rails.configuration.webpack.dev_server.port)
+            ::Rails.configuration.webpack.dev_server.manifest_host,
+            ::Rails.configuration.webpack.dev_server.manifest_port)
           http.use_ssl = ::Rails.configuration.webpack.dev_server.https
           http.verify_mode = OpenSSL::SSL::VERIFY_NONE
           http.get(dev_server_path).body
```

Comment:
if there's a problem fetching the manifest, the below error message doesn't look right.. should mention the manifest host/port.